editor/src/components/code-editor/code-editor-container.tsx

```diff
@@ -16,6 +16,10 @@ const VSCodeIframeContainer = betterReactMemo(
     const url = new URL(baseIframeSrc)
     url.searchParams.append('project_id', projectID)
 
+    const branchName = url.searchParams.get('branch_name')
```

Comment:
there's a `getEditorBranchNameFromURL` helper function, currently used by no one. I think it's better to use a helper function so we don't have to hunt for 'branch_name' in the codebase

Reply:
Agreed, or at least replace it was a const please

Reply:
Fixed in c96f554a.